surefire-its/src/test/resources/settings.xml

```diff
@@ -17,13 +17,37 @@
   ~ specific language governing permissions and limitations
   ~ under the License.
   -->
-<settings xmlns="http://maven.apache.org/SETTINGS/1.1.0"
-          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd">
-    <localRepository>${maven.repo.local}</localRepository>
-    <interactiveMode>false</interactiveMode>
-    <pluginGroups>
-        <pluginGroup>org.apache.maven.plugins</pluginGroup>
-        <pluginGroup>org.codehaus.mojo</pluginGroup>
-    </pluginGroups>
+<settings>
```

Comment:
This file is related to `maven-invoker-plugin`. The right fix would be the one I removed:

```
<plugin>
        <artifactId>maven-help-plugin</artifactId>
        <version>2.2</version>
        <executions>
          <execution>
            <id>settings.xml</id>
            <phase>generate-test-resources</phase>
            <goals>
              <goal>effective-settings</goal>
            </goals>
            <configuration>
              <output>${project.build.directory}/private/settings.xml</output>
              <showPasswords>${it.settings.showPasswords}</showPasswords>
            </configuration>
          </execution>
        </executions>
      </plugin>
```

I will introduce it back t project and check it out after the file `settings.xml` was generated.